packages/chain-adapters/src/utxo/UtxoBaseAdapter.ts

```diff
@@ -323,12 +324,15 @@ export abstract class UtxoBaseAdapter<T extends UtxoChainId> implements IChainAd
 
       const signTxInputs: BTCSignTxInput[] = []
       for (const input of inputs) {
-        if (!input.path) continue
+        // TOOD(gomes): v. dangerous, narrow this monkey patch for Phantom only
+        // if (!input.path) continue
```

Comment:
@kaladinlight to confirm the sanity of this - this doesn't inject `wallet` so we can't narrow down on `instanceof PhantomHDWallet` - is this just a paranoia check or is removing it indeed very dangerous?

Reply:
Removed here: https://github.com/shapeshift/web/pull/7766/commits/6b4467ad9c03f67b97e603f80c52c8bcb9dccddb

it was more just type narrowing as utxos by xpub always returned a path. updated to default to bip44Params if no path is present.